### ISSUE TITLE
Localization

### ISSUE THREAD

**Issue by airon90** [ccomeaux/boardgamegeek4android#115]
~~Hi, I would like to translate the app. How can I do that?
Moreover,~~ I would like that game titles are shown up in my native language. Is it possible to add such options?

**Comment by Vinne2** [comment]
I think it would be easier for the developer to manage the issues if you make two separate issues – 1 for helping in translation and 1 for localization enhancement. I suggest editing this issue here with leaving only "localization" here (because I've already replied on this, above) and making a new issue with translation offer. (I admit that I could also help in translation so I'd reply there, too). Cheers!
P.S.
And please specify in title what localization do you mean – because there's also a localization when logging a play ;) The issue will be easier to find on the list :)

**Comment by airon90** [comment]
>  And please specify in title what localization do you mean – because there's also a localization when logging a play ;) The issue will be easier to find on the list :)

That is not true: example, when I look for "Intrigo a palazzo" it shows me the Italian title of the game in italics, then when I select the game the English original title is shown. It is correct that, as an [add your origin here], I see the localized title of the games.

**Comment by Vinne2** [comment]
Ah, now I see. You mean that you see: 
- local title (in italics) in search results,
- but original title on game page (game info tab).
Now I understand. So, if the app already knows the local title I think it should not be tough job to implement that also on game page.

**Comment by airon90** [comment]
Instead I think that this issue would be maybe more complicated that it could be.

1. What if I, Italian, buy both English and Italian version of a board game, e.g. Cards Against Humanity?
2. What if I, Italian, buy both English and Italian version of a board game whose title is completely different from the original one, e.g. (game 26208)[https://boardgamegeek.com/boardgame/26208]?

I think that we could do something, e.g. a setting to show the title of the game in a chosen language, but not a complete localization as of upstream limitations.

**Comment by airon90** [comment]
Translation bug is filed in #119

**Comment by ccomeaux** [comment]
@airon90 What do you see in the website for these cases? When you view the CArds Against Humanity page at https://www.boardgamegeek.com/boardgame/50381, do you see the game in Italian? I don't know that I'll be able to handle it any better in the app than BGG does.

**Comment by airon90** [comment]
> do you see the game in Italian?

No, as in Italy we use to call CaH "Cards against humanity", not "Carte contro l'umanità"

**Comment by Vinne2** [comment]
For example me and the game *7 Wonders*, in Polish *7 Cudów Świata*.

I have this game in my collection ([see here](https://boardgamegeek.com/collection/user/Kaligula?rankobjecttype=subtype&rankobjectid=1&columns=title%7Cthumbnail%7Cstatus%7Cversion%7Crank&minrating=8&rating=8&minbggrating=7&bggrating=8&minplays=8&geekranks=Board%20Game%20Rank&excludesubtype=boardgameexpansion&own=0&trade=0&want=0&wanttobuy=0&prevowned=0&comment=1&wishlist=0&rated=1&played=1&wanttoplay=0&preordered=0&hasparts=0&wantparts=0&objecttype=thing&ff=1&subtype=boardgame)) with chosen Polish 2016 edition. I don't know how the link displays on your computer but I can see Polish name in Title column (*7 Cudów Świata*).

In the BGG app, while on collection list, I can see the same Polish name *7 Cudów Świata*. BUT when I click the game to see its details I can see the original English name *7 Wonders*. On *My games* tab in the details of my collection item I can see Polish name *7 Cudów Świata*. I also see Polish name in Collection Details (each – Designers/Artists/Publishers/Mechanics/Categories).

And in each other place I checked I see English name – collection Item details (as mentioned above), Stats (Play stats via Item details or Play stats via Menu), the list of Plays (regardless of how I enter it – via item Details, via Plays, Players, Locations or the GeekBuddies).

It would be way easier to see the localized name, as there are games with completely different name in English than localized name (e.g. [**en**: "*Wheedle*" meaning *swingle/plead* vs. **pl**: "*Bankrut*" meaning *bankrupt person* vs. **de**: "*GmbH & Co.KG*" meaning *Inc. (suffix after corporation's name)*](https://boardgamegeek.com/boardgame/5393/wheedle/versions?pageid=1)).

![Screenshot_20200420-182010](https://user-images.githubusercontent.com/3950811/79776382-f4b17f80-8335-11ea-9610-7594ef1f0af2.png)

███████████████████████████

![Screenshot_20200420-182017](https://user-images.githubusercontent.com/3950811/79776530-2cb8c280-8336-11ea-94b9-83f264787006.png)

███████████████████████████

![Screenshot_20200420-182023](https://user-images.githubusercontent.com/3950811/79776544-30e4e000-8336-11ea-801b-29d279df47da.png)

███████████████████████████

![Screenshot_20200420-182111](https://user-images.githubusercontent.com/3950811/79776549-32aea380-8336-11ea-861a-8fb21cd6443a.png)

███████████████████████████

![Screenshot_20200420-182909](https://user-images.githubusercontent.com/3950811/79776553-33dfd080-8336-11ea-80db-7725de7cc251.png)

███████████████████████████

![Screenshot_20200420-182039](https://user-images.githubusercontent.com/3950811/79776914-be283480-8336-11ea-888b-d65263632c2f.png)

███████████████████████████

![Screenshot_20200420-182043](https://user-images.githubusercontent.com/3950811/79776612-4ce88180-8336-11ea-9c52-793229075de9.png)

███████████████████████████

![Screenshot_20200420-183045](https://user-images.githubusercontent.com/3950811/79776622-4f4adb80-8336-11ea-8c29-d0732c189ea0.png)
